data_validation/validation_builder.py

```diff
@@ -90,12 +95,21 @@ def get_metadata(self):
 
     def get_group_aliases(self):
         """ Return List of String Aliases """
-        return self.group_aliases.keys()
+        return list(self.group_aliases.keys())
+
+    def get_primary_keys(self):
+        """ Return List of String Aliases """
+        # do we need this?
```

Comment:
Don't think we need this since you can always use config_manager.primary_keys()

Reply:
In principal I agree but a lot of inconsistencies have sprung up between the validation builder and config manager where the two objects emit different datatypes of the same business objects. Most references in that method use the validation builder so I did the same. I'd like to propose a cleanup sprint sometime in the future to make the separation of functionality more explicit